### DISCUSSION TITLE
Does AtlasMap GUI supports to add new property

### DISCUSSION THREAD

**Discussion by gurumurthy-godlaveeti** [atlasmap/atlasmap#4142] [Q&A]
Hi Team,

I have scenario to add new property under source complex object. Suppose, Lets observe below screenshot & consider it as Release1.
![image](https://user-images.githubusercontent.com/101185013/176639574-9772d68f-8c23-430c-b88e-93813e70b5f3.png)

Part of Release2, we introduced new field called "phoneNumber" which needs to be added in both source & target side. Does it needs to changed in source & target structures & reimporting again?

While reimporting, we loose the mapping of existed fields. When source & target is having huge properties like 100 or more then its big job to make mapping again so wish to check possibility.

**Comment by igarashitm** [comment]
There are open issues for that - https://github.com/atlasmap/atlasmap/issues/1417, https://github.com/atlasmap/atlasmap/issues/3353
Contribution is always welcome.
